wsl-builder/common/releasesinfo.go

```diff
@@ -232,3 +233,16 @@ func (w *WslReleaseInfo) refreshedTerminalProfileID() error {
 
 	return nil
 }
+
+func RootfsUrl(r WslReleaseInfo, arch string) string {
```

Comment:
shouldn’t RootfsUrl (I’m unsure if the linter won’t force it to URL btw), be a method on WSLReleaseInfo?

Reply:
Fair point. Changing it.